config/locales/controllers/admin/fr.yml

```
@@ -0,0 +1,5 @@
+---
+en:
```

```suggestion
fr:
```

(bug / blocking) - I think this is why all the confirmation messages are coming up in French for me, even though my Forem setting is for English

---

Urgggh crap thanks!